release.yaml

```diff
@@ -3,30 +3,18 @@ elemental:
 elemental-crd:
   - 103.1.0+up1.4.2
 fleet:
-  - 102.2.2+up0.8.2
   - 103.1.1+up0.9.1-rc.6
 fleet-agent:
-  - 102.2.2+up0.8.2
   - 103.1.1+up0.9.1-rc.6
 fleet-crd:
-  - 102.2.2+up0.8.2
   - 103.1.1+up0.9.1-rc.6
 harvester-cloud-provider:
-  - 102.0.2+up0.2.3
   - 103.0.1+up0.2.3
 harvester-csi-driver:
-  - 102.0.2+up0.1.17
   - 103.0.1+up0.1.17
-neuvector:
-  - 102.0.7+up2.7.1
```

Comment:
Removing the already released version of neuvector `102.0.7+up2.7.1` which was not in this release